lega_init/configure.py

```diff
@@ -153,8 +154,22 @@ def add_conf_key(self, file_name, armor=True):
         If a passphrase is not provided it will be generated.``
         """
         pub, sec = self.pgp_pair
+
+        pkey, skey = self.c4gh_pair
+
+        # os.umask(0o222)  # Restrict to r-- r-- r--
+        with open(self._config_path + f'/{file_name}.c4gh.pub', 'bw', ) as f:
```

Comment:
Check the comma after the file opening mode is set

Reply:
Done. thx.